plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -383,11 +378,11 @@ public LibvirtUtilitiesHelper getLibvirtUtilitiesHelper() {
     }
 
     public CPUStat getCPUStat() {
-        return _cpuStat;
+        return new CPUStat();
```

Comment:
Is this coded on purpose? I mean, can we use it as a stateless call? At first sight this `getCPUStat` seem to return an object property, but instead it will always return a different object instance.